core/src/main/java/org/mobilitydata/gtfsvalidator/parsing/RowParser.java

```diff
@@ -203,10 +206,19 @@ public void checkRowColumnCount(CsvFile csvFile) {
   public String asString(int columnIndex, boolean required) {
     String s = row.asString(columnIndex);
     if (required && s == null) {
-      addErrorInRow(
+      addNoticeInRow(
           new MissingRequiredFieldError(
               row.getFileName(), row.getRowNumber(), row.getColumnName(columnIndex)));
     }
+    if (s != null) {
+      final String trimmed = s.strip();
```

Comment:
Nit - given that there is also a Java method `trim()`, this variable is probably better named `stripped`.

Reply:
Done.

By the way, can you tell the difference between "strip" and "trim" as a native English speaker? To me, these words are interchangeable and one must check the documentation to understand the difference :)

Reply:
Actually, that change belongs to another pull request. Fixed.

https://github.com/MobilityData/gtfs-validator/pull/729

Reply:
>By the way, can you tell the difference between "strip" and "trim" as a native English speaker? 

Not in the context of Java :). In general use "strip" has somewhat of a more aggressive and coarse connotation than "trim". "trim" implies a more precise and neat action.